datastore/tasks.rb

```diff
@@ -48,14 +49,12 @@ def mark_done task_id
 
 # [START retrieve_entities]
 def list_tasks
-  query = @datastore.query("Task").
-          order("created")
-
+  query = @datastore.query("Task").order("created")
```

Comment:
Thanks!  Makes sense.

Usually we do that when we go over 80 LOC.

Once you go over 80, sometimes it can be nicer to put one on each line because it looks nicer than having 3 on one line with 1 trailing on a newline, for example.